checks/src/test/java/com/kozaxinan/android/checks/NetworkLayerClassSerializedNameDetectorTest.kt

```diff
@@ -30,7 +30,7 @@ internal class NetworkLayerClassSerializedNameDetectorTest {
     lint()
         .files(
             retrofit(),
-            gson(),
+            annotation(),
```

Comment:
this test belongs to different lint rule. Can you revert this part and keep changes in new files.

Reply:
I'll revert the changes on that class 👍